lib/ret_web/channels/auth_channel.ex

```diff
@@ -22,17 +23,27 @@ defmodule RetWeb.AuthChannel do
     if !Map.get(socket.assigns, :used) do
       socket = socket |> assign(:used, true)
 
-      # Create token + send email
-      %LoginToken{token: token, payload_key: payload_key} = LoginToken.new_login_token_for_email(email)
+      if can?(nil, create_account(nil)) || Account.exists_for_email?(email) do
```

Comment:
Hmm, seems a bit odd to shoe-horn this in to canada if it's really just an app config check. Maybe we should just do something like `AppConfig.feature_off(:disable_sign_up)`, if we end up doing more of this in the future.

Reply:
I had originally had this as a config check, but it did seem to me to be about access control. The permission basically encapsulates security policy defined by the app configuration. I think probably we may need to carve out these kinds of configs separate from `feature` configs, or at least subcategorize them.

Another reason I moved away from the config check is because the next step here is going to be writing an API for managing accounts (the full use case is locking down the IEEEVR server.) From that perspective, the effective check would need to be in two places (unless we want to add a 'model tier') so it felt like a permission check in those two places was more reasonable than a config check to enforce authorization.

Reply:
Oh, also one more thing. The check here is actually a pre-emptive check that doesn't actually enforce policy per se: it just doesn't send the email as a "convenience", since if a user got an email, they'd be denied the ability to create the account if a new account was needed, even if they had a valid login token. So technically we could remove this check with no loss of security.

Reply:
(in that sense, a config check would be less weird to put here vs the check later during account creation, since it's more of a "user interface" mask, not a policy enforcement check)